x-pack/plugins/watcher/public/application/sections/watch_list/components/watch_list.tsx

```diff
@@ -409,29 +422,43 @@ export const WatchList = () => {
     };
 
     content = (
-      <EuiInMemoryTable
-        items={availableWatches}
-        itemId="id"
-        columns={columns}
-        search={searchConfig}
-        pagination={PAGINATION}
-        sorting={true}
-        selection={selectionConfig}
-        isSelectable={true}
-        message={
-          <FormattedMessage
-            id="xpack.watcher.sections.watchList.watchTable.noWatchesMessage"
-            defaultMessage="No watches to show"
-          />
-        }
-        rowProps={() => ({
-          'data-test-subj': 'row',
-        })}
-        cellProps={() => ({
-          'data-test-subj': 'cell',
-        })}
-        data-test-subj="watchesTable"
-      />
+      <div data-test-subj="watchesTableContainer">
```

Comment:
The search box in the table doesn't appear to be able to accept a `data-test-subj` prop, so we have to access it via a CSS selector in the tests. In order to use a CSS selector, you need to first target the closest test subject.